src/pages/common/_model_page.c

```diff
@@ -250,7 +250,7 @@ static const char *protoselect_cb(guiObject_t *obj, int dir, void *data)
         // Load() the new protocol
         Model.protocol = new_protocol;
         Model.radio = PROTOCOL_GetRadio(new_protocol);
-        Model.tx_power = RADIO_TX_POWER_COUNT[Model.radio]-1;
+        if (Model.radio < TX_MODULE_LAST) Model.tx_power = RADIO_TX_POWER_COUNT[Model.radio]-1;
```

Comment:
curious, why Model.radio has an invalid value? why not fix the place which cause invalid value instead of check here? The check is more or less a hack here.

Reply:
I agree, and we had a couple comments on this in the original PR.  Have not yet come up with a cleaner way to handle the relationship between models, protocols, internal radios, and external radios, that doesn't add a lot of code.  I will try to clean this up but am traveling the next 10 days. Would be better to merge as master is writing invalid power values now to the model files if the protocol doesn't have an associated radio.  Doesn't cause a problem because the value is ignored when read.  The emulator crashes when trying to write.